src/store/modules/product/mutations.ts

```diff
@@ -20,6 +20,10 @@ const mutations: MutationTree <ProductState> = {
   [types.PRODUCT_CLEAR_UPLD_PRDTS] (state) {
     state.uploadProducts = {};
   },
+  [types.PRODUCT_CLEAR_SEARCH_PRDTS] (state) {
```

Comment:
We don't need to create another mutation to clear search list, we can use the `PRODUCT_SEARCH_UPDATED` mutation to clear the search product by passing empty values to the products and totalProductsCount.

Reply:
Improved code please take a look Sir.